exercises/exercisesModelsTest.spec.js

```diff
@@ -0,0 +1,9 @@
+const exercise = require('./exercisesModels');
+
+describe('get single exercise', () => {
+  it('is able to get a single exercise by ID from the db!', async () => {
+    const singleExercise = await exercise.findExerciseById('799');
+    expect(singleExercise.id).toBe(799);
+    expect(singleExercise.exercise_name).toBe('Knee Across The Body');
+  });
+});
```

Comment:
This test won't pass, server has not been imported and supertest is missing.

Reply:
This test passes, calls exercise by ID returns object containing things its supposed to